src/main/frontend/components/editor.cljs

```diff
@@ -382,12 +382,20 @@
     (set-up-key-down! repo state format)
     (set-up-key-up! state input input-id search-timeout)))
 
-(defn- get-editor-heading-class
-  [content heading-level]
-  (if (string/includes? content "\n")
```

Comment:
There's a problem when a block's content has a heading followed by some other lines,  for example:
```
## heading
some content
```
You can see the problem at https://www.loom.com/share/35f6c68234d04e2aaf7fbf54cdbc8214

Reply:
Ah good catch! Sorry I removed that. I'll fix it.

Reply:
No worries at all, thanks!

Reply:
Fixed!